packages/bundler-plugin-core/src/plugins/react-annotate-plugin.js

```diff
@@ -0,0 +1,480 @@
+/**
+ * This plugin is vendored in from FullStory's Babel React Annotate plugin:
+ * https://github.com/fullstorydev/fullstory-babel-plugin-annotate-react/tree/master
```

Comment:
h: Vendoring in requires us to copy paste license.

If you make the review changes recommended we should also add notes about what things we changed.